dashboard/app/views/layouts/_header.html.haml

```diff
@@ -49,7 +49,7 @@
 
   sign_in_user = current_user || user_type && OpenStruct.new(
     id: nil,
-    short_name: I18n.t("nav.user.#{user_type}"),
```

Comment:
`nav.user.student` and `nav.user.teacher` were missing from en.yml which is why users sometimes saw the "missing translation" message in the header.